eclipse-theia/theia-blueprint
# Update to Theia 1.16.0

#### What it does

This updates Blueprint to Theia v1.6.0, the latest released version of the framework Blueprint is based-on. 

#### How to test

Build PR and check that it still builds and runs, both "in-place" and in packaged form.

#### Review checklist


#### Reminder for reviewers

- as a reviewer, I agree to behave in accordance with [the review guidelines](https://github.com/theia-ide/theia/blob/master/doc/pull-requests.md#reviewing)
